spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/RuntimeAppsController.java

```diff
@@ -90,6 +90,16 @@ public AppStatusResource display(@PathVariable String appId) {
 		return statusAssembler.toResource(status);
 	}
 
+	@RequestMapping("/logs/{streamName}")
```

Comment:
We need to figure out a different path as `http://localhost:9393/runtime/apps/logs/ticktock` would not work as we already use `http://localhost:9393/runtime/apps/ticktock.time-v1`. Either something in `/runtime/logs/` and/or `/runtime/apps/ticktock.time-v1/logs`.

Reply:
yeah, it makes sense. I will update the PR